app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -126,10 +151,19 @@ class ProjectActivity : BaseActivity<ProjectViewModel.ViewModel>() {
             this.viewModel.inputs.heartButtonClicked()
         }
 
+        native_back_this_project_button.setOnClickListener {
+            this.viewModel.inputs.nativeBackProjectButtonClicked()
+        }
+
         manage_pledge_button.setOnClickListener {
             this.viewModel.inputs.managePledgeButtonClicked()
         }
 
+        rewards_toolbar.setNavigationOnClickListener {
+            this.viewModel.inputs.hideRewardsFragmentClicked()
+            this.project_toolbar.visibility = View.VISIBLE
```

Comment:
i don't think we need to set this anymore. the elevation of the card is above the toolbar